### PR TITLE
feat: Add support for Live Activity with ActivityKit push notifications

### PR DESCRIPTION
**Pull request by dplewis**
https://developer.apple.com/documentation/activitykit/updating-and-ending-your-live-activity-with-activitykit-push-notifications

This PR also fixes the broken tests.

### CODE REVIEW

**Comment by dplewis** [comment]
@mtrezza thanks for fixing the CI, is this good?

**Review by mtrezza — APPROVED** [review]
Looks good to me; not sure if we can find anyone to review this, since it's only a part of the integration as I understand. In any case we could merge it and fix any issues that may arise.

**Comment by dplewis** [comment]
Its not specific really, adding 5 fields to a push notification. I don't think apple would introduce a breaking change over this

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.2.0](https://github.com/parse-community/node-apn/releases/tag/5.2.0)

**Comment by dplewis** [comment]
@mtrezza I don't know how but I have merge rights to the original repo but not this one

File: lib/notification/apsProperties.js [hunk 1]
```diff
@@ -83,6 +83,36 @@ module.exports = {
...
+
+  set staleDate(value) {
+    if (typeof value === 'number' || value === undefined) {
+      this.aps['stale-date'] = value;
+    }
+  },
+
+  set event(value) {
+    if (typeof value === 'string' || value === undefined) {
+      this.aps.event = value;
```
Comment (alanwguo) [inline lib/notification/apsProperties.js:106]: Is this supposed to be `events`? The docs here refers to the key as "events": https://developer.apple.com/documentation/activitykit/updating-and-ending-your-live-activity-with-activitykit-push-notifications

Comment (dplewis) [reply lib/notification/apsProperties.js:106]: Good catch!

**Pull request merged**